Cargo.toml

```diff
@@ -239,6 +241,7 @@ tracing = "0.1"
 tracing-opentelemetry = "0.27"
 tracing-subscriber = { version = "0.3.18", features = ["env-filter"] }
 uuid = { version = "1.0", features = ["v4", "serde"] }
+zeroize = { version = "1.8.1", features = ["derive"] }
```

Comment:
Yes because `strata-cli` uses it and it has a `derive(ZeroizeOnDrop)`.
And I've moved `zeroize` to the root-level `Cargo.toml` and added a `workspace = true` in `strata-cli`.

Forgot to mention.

Reply:
Ah, I missed that.